test/integration/tox.ini

```diff
@@ -0,0 +1,14 @@
+[tox]
```

Comment:
How do you get tox to call pip twice with different reqs? Or I suppose we could call it manually to update ansible after tox, though that seems odd to me...